persistence/sql/persister_session.go

```diff
@@ -36,10 +36,29 @@
 		return nil, err
 	}
 
+	devices, err := p.GetSessionDevices(ctx, sid)
```

Comment:
It makes probably more sense to use `.Eager()` for this. For the identity we don't use Eager() because it has some custom validation built in.

You can pass arguments to eager which say what relation you want to be hydrated. Saves you some code :)